prediction_market_agent/agents/known_outcome_agent/benchmark.py

```diff
@@ -53,9 +53,8 @@ def __init__(
         model: str,
         max_tries: int,
     ) -> None:
-        self.model: str = model
         self.max_tries = max_tries
-        super().__init__(agent_name=agent_name, max_workers=max_workers)
+        super().__init__(agent_name=agent_name, max_workers=max_workers, model=model)
```

Comment:
After https://github.com/gnosis/prediction-market-agent-tooling/commit/10870b51eb70668fb0363b90fbbf496ff10e6135 `model` is being overridden to `None`